kits/java/src/main/java/com/luxai/lux/Board.java

```diff
@@ -0,0 +1,31 @@
+package com.luxai.lux;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.ArrayList;
+import java.util.Map;
+
+public class Board {
+    // step = 0
+    @JsonProperty("rubble_init")
+    public int[][] rubble;          // Mixed up X and Y. I don`t know it is my bad or json-feature

```

Comment:
You did not mix up X and Y. There was a change (v1.1.0) which switched from [y][x] indexing to [x][y]. So you're all good.

Reply:
updated